lib/registry.js

```diff
@@ -244,6 +244,14 @@ Registry.prototype.configureModel = function(ModelCtor, config) {
       modelName);
   }
 
+  var hasNewMethods = config.methods && Object.keys(config.methods).length;
+  var hasDescendants = this.getModelByType(ModelCtor) !== ModelCtor;
+  if (hasNewMethods && hasDescendants) {
+    g.warn(
+      'Child models of `%s` will not inherit newly defined remote methods %j.',
+      modelName, config.methods);
```

Comment:
I think dumping the whole method-definition object will clutter the output. I am proposing to print only method names?

```js
g.warn(
  'Child models of `%s` will not inherit newly defined remote methods %j.',
  modelName, Object.keys(config.methods));
```

To avoid calling `Object.keys` twice, it's better to store it in a variable, e.g. `newMethodNames`.